### DISCUSSION TITLE
Find GitHub login from SCIM ID

### DISCUSSION THREAD

**Discussion by groboclown** [community/community#50873] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I'm able to query the SCIM API for [listing provisioned users](https://docs.github.com/en/enterprise-cloud@latest/rest/scim?apiVersion=2022-11-28#list-scim-provisioned-identities), and I'm able to query the [organization's members](https://docs.github.com/en/enterprise-cloud@latest/rest/orgs/members?apiVersion=2022-11-28#list-organization-members), but I do not see any way to connect the two to perform an audit of users.  The web UI for the people in an organization shows that GitHub has that information, but I don't see how to obtain it through the REST API.  Is there an endpoint that either retrieves the GitHub login from any of the SCIM identifiers?  I'm guessing the github ID is the most likely choice.

**Comment by Anmol-Baranwal** [comment]
> To search for filtering conflicting pull requests in a GitHub repository, you can refer to the official documentation [here](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests). 

However, note that there is no direct way to find a conflicting pull request unless the reviewer has labeled it with a specific label. You can use the search query `label:LABEL` to filter pull requests with a specific label, or use `status:failure` to filter pull requests with failed checks depending on the repository's configuration.

**Comment by groboclown** [comment, marked as answer]
Looks like the answer is vai GraphQL:

https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/troubleshooting-identity-and-access-management-for-your-organization#using-graphql

**Reply by dannyrb** [reply]
Thanks! This was a real life saver to find ^_^
